search/templates/search/search.html

```diff
@@ -193,6 +170,26 @@ <h5 class="modal-title" id="mapModalLabel">Search Result:</h5>
           </div>
         </div>
 
+
+        {% if search_data.stats %}
+    <p>
+        <hr class="mt-0 mb-2 mt-2">
+        <div class="horizontal-list">
+        <h5>Complaints Level:</h5>
+        <span style="margin-left: 0.5rem; color: {{search_data.css_color_for_complaint_level}};">
+            {{ search_data.description_for_complaint_level }}
+        </span>
+        <ul id="stats-list" class="horizontal-list" stype="display: inline-block">
+            {% for key, percentage in search_data.stats %}
+            <li>
+                <span>{{ key }} </span><div class="meter"><span style="width: {{ percentage }}%"></span></div>
+            </li>
+            {% endfor %}
+        </ul>
+        <a href="{% url 'data_311'%}?zip_code={{zipcode}}">See more</a>
+        </div>
+    </p>
+    {% endif %}
```

Comment:
It appears the the price range is no longer being displayed on the search page cards.

from `search_styling` branch:
![Screen Shot 2019-12-08 at 7 16 52 PM](https://user-images.githubusercontent.com/32490273/70399113-6aecbf80-19ef-11ea-9bc6-cfd5c53f7d3a.png)

from develop-branch:
![Screen Shot 2019-12-08 at 7 16 57 PM](https://user-images.githubusercontent.com/32490273/70399126-72ac6400-19ef-11ea-8895-121d6caae077.png)